src/main/webapp/itinerary-overview.html

```diff
@@ -19,28 +19,66 @@
             href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
     />
 </head>
-<body onload="init(); displayMap()">
```

Comment:
Why is the call from `init()` being removed? I think we still need it for initializing the navbar.

Reply:
Those elementID's don't exist on my page anymore because I removed the header and footer so there is nothing to initialize because they won't show. It would just throw an error.